pkg/ffuf/config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	OutputFile             string                    `json:"outputfile"`
 	OutputFormat           string                    `json:"outputformat"`
 	IgnoreBody             bool                      `json:"ignorebody"`
+	NoBanner               bool                      `json:"nobanner"`
```

Comment:
In case we go with writing to `stderr`, we don't need to add a new configuration variable.